bakery/src/tasks/build-pdf.js

```diff
@@ -24,7 +24,9 @@ const task = () => {
           dedent`
           exec 2> >(tee artifacts/stderr >&2)
           book_dir="mathified-book/$(cat book/collection_id)"
-          prince -v --output="artifacts/$(cat book/pdf_filename)" "$book_dir/collection.mathified.xhtml"
+          pdf_filename="$(cat book/collection_id)-$(cat book/version)-${'${book/server_name}'}-$(cat book/job_id).pdf"
```

Comment:
Maybe I'm wrong, but does this work with `${book/server_name}` like this? You don't need to do something like `$(cat book/server_name)`?

Reply:
you're correct on this, awesome catch, thank you